tests/test_browser_custom_keybinds.py

```
@@ -13,14 +13,51 @@
+    request_id: str = "1",
+) -> Any:
+    payload = {
+        "id": request_id,
+        "method": method,
+        "params": params if params is not None else {},
+    }
+    raw = client._send_command(json.dumps(payload))
+    try:
+        parsed = json.loads(raw)
+    except json.JSONDecodeError as exc:
+        raise RuntimeError(f"Invalid v2 JSON response for {method}: {raw}") from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)